lisa-utils/src/main/scala/lisa/utils/KernelHelpers.scala

```diff
@@ -201,23 +201,24 @@ trait KernelHelpers {
   given Conversion[String, Term] = FOLParser.parseTerm(_)
   given Conversion[String, VariableLabel] = s => VariableLabel(if (s.head == '?') s.tail else s)
 
-
   given Conversion[Term, LambdaTermTerm] = LambdaTermTerm(Seq(), _)
+  given Conversion[Formula, LambdaTermFormula] = LambdaTermFormula(Seq(), _)
```

Comment:
Should we provide an implicit conversion like this? 

Whether we map it to use empty Seq or use free variables, it is likely that the user will see `Requirement failed: incorrect number of arguments` instead of the current `Expected: LambdaTermFormula Received: Formula`, which is definitely more helpful.

Reply:
I see now that you added the `lambda` utilities down the line, which make this work better, so nevermind!